packages/asset-swapper/CHANGELOG.md

```diff
@@ -21,609 +21,322 @@ CHANGELOG
 
     * Fantom deployment (#347)
 
-## v16.29.3 - _October 18, 2021_
-
-    * Update neon-router version and address breaking changes (#344)
-
-## v16.29.2 - _October 13, 2021_
-
-    * Check MAX_IN_RATIO in sampleBuysFromBalancer (#338)
-    * Go back to using transformERC20 (instead of transformERC20Staging) (#343)
-
-## v16.29.1 - _October 4, 2021_
-
-    * Remove `Clipper` as a custom liquidity source (#335)
-
-## v16.29.0 - _October 4, 2021_
-
-    * Initial integration of neon-router (behind feature flag) (#295)
-
-## v16.28.0 - _September 29, 2021_
-
-    * Update ExchangeProxySwapQuoteConsumer for Multiplex V2 and friends (#282)
-
-## v16.27.5 - _Invalid date_
-
-    * Remove protocol fees by setting `PROTOCOL_FEE_MULTIPLIER` to 0 (#333)
-
-## v16.27.4 - _September 15, 2021_
-
-    * Dependencies updated
-
-## v16.27.3 - _September 14, 2021_
-
-    * Dependencies updated
-
-## v16.27.2 - _September 14, 2021_
-
-    * Dependencies updated
-
-## v16.27.1 - _September 8, 2021_
-
-    * Fix ApproximateBuys sampler to terminate if the buy amount is not met (#319)
-
-## v16.27.0 - _September 1, 2021_
+## v6.7.0 - _September 1, 2021_
```

Comment:
not sure what happened to this file, but you should revert all these changes

Reply:
Yeah this file gets automatically generated from `CHANGELOG.json`, so no need to ever have any changes in a `CHANGELOG.md` file.

Reply:
This seems not resolved?